config/bootstrap.php

```diff
@@ -99,6 +103,15 @@
 date_default_timezone_set('UTC');
 
 /**
+ * Sets the default output timezones.
+ * This output timezone is also being used by TimeHelper.
+**/
+Time::setDefaultOutputTimezone(Configure::read('App.defaultOutputTimezone'));
+Date::setDefaultOutputTimezone(Configure::read('App.defaultOutputTimezone'));
+FrozenTime::setDefaultOutputTimezone(Configure::read('App.defaultOutputTimezone'));
+FrozenDate::setDefaultOutputTimezone(Configure::read('App.defaultOutputTimezone'));
```

Comment:
IMHO its good if you have an application where there are nu user specific preferences but the timezone, locale, date/money/number formats are set globally. It is easy and convenient and you can always later change it or remove the line in bootstrap. IMHO.